workflowitems.md

```diff
@@ -149,9 +149,15 @@ If fails the following status code are expected
 Multipart POST request will typically result in the creation of a new file in the section identified by the name of the variable used for the upload (uploads is the default name of the user uploaded content). The process will be managed by the implementation bind with the identified section.
 If succeed a 201 code will be returned and the new state of the workflowitem serialized in the body
 
-## DELETE Method 
+## DELETE Method
+**DELETE /api/submission/workflowitems/<:id>**
+
 Reset a workflow sending back the item to the workspace regardless to the step reached.
 
-204 No content - if the operation succeed
-403 Unauthorized - if you are not loggedin as an administrator
-404 Not found - if the workflow doesn't longer exist
+If an optional parameter `?expunge=true` is included, the workflow item is deleted instead of being returned to the workflow.
```

Comment:
It seems like the permissions to expunge versus sending back to a workspace are going to be different here.  So, we may want to call that out more explicitly.  My assumption is anyone in the proper workflow group can do a DELETE (without expunge), as that's essentially just a workflow rejection.  But, *only* an Admin or someone with Item DELETE privileges can use the `expunge=true` option.

Is that correct?  Or am I missing something here.

Reply:
No, both DELETE solutions are admin-only features (the previous version of the contract also defined it was admin-only).

Workflow members can only perform https://github.com/DSpace/Rest7Contract/blob/master/claimedtasks.md#post-method-single-resource-level, a POST with submit_reject=true is a workflow rejection.

Reply:
Thanks for clarifying, @benbosman , and for the minor update to the Contract.  This looks good to me now